pipelines/mix/mix.py

```diff
@@ -0,0 +1,174 @@
+import os
+import logging
+from time import time
+from settings import base
+from generics import pipeline as p, task as t
+
+from pipelines.mix.tasks import (
+    get_mixed
+)
+
+
+logger = logging.getLogger('LCTK_APPLICATION_LOGGER')
+
+
+class MixPipeline():
```

Comment:
Naming things are hard. Is there a more descriptive name for this pipeline?
Like ResidentialCommercialAnalysisPipeline? <- not literally this, but you get the idea

Reply:
changing it to MixedFeederPipeline

Reply:
This is the commit: https://github.com/slacgismo/loadinsight/commit/7dbbdec84b5ea2dfb9a756f0e514ec9cb258fb07